### ISSUE TITLE
[Wakanim] 

### ISSUE THREAD

**Issue by Totorrr** [yt-dlp/yt-dlp#1609]
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2021.11.10.1**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Description

On the wakanim website, I have trouble downloading videos, for example: 

```
$ yt-dlp 'https://www.wakanim.tv/fr/v2/catalogue/episode/27775/fruits-basket-saison-3-episode-13-vostfr'    
[Wakanim] 27775: Downloading webpage
ERROR: [Wakanim] 27775: Unable to extract manifest url; please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.; please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
```

It looks like the website has changed and yt-dlp is not working with it anymore. 

Please see the log file. 

### Verbose log

```shell
[debug] Command-line config: ['-Uv', 'https://www.wakanim.tv/fr/v2/catalogue/episode/27775/fruits-basket-saison-3-episode-13-vostfr']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, err UTF-8, pref UTF-8
[debug] yt-dlp version 2021.11.10.1
[debug] Python version 3.7.3 (CPython 64bit) - Linux-5.10.0-0.bpo.8-amd64-x86_64-with-debian-10.11
[debug] exe versions: ffmpeg 4.3.1-8, ffprobe 4.3.1-8
[debug] Optional libraries: Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
yt-dlp is up to date (2021.11.10.1)
[debug] [Wakanim] Extracting URL: https://www.wakanim.tv/fr/v2/catalogue/episode/27775/fruits-basket-saison-3-episode-13-vostfr
[Wakanim] 27775: Downloading webpage
ERROR: [Wakanim] 27775: Unable to extract manifest url; please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.; please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/yt_dlp/extractor/common.py", line 595, in extract
    ie_result = self._real_extract(url)
  File "/usr/lib/python3/dist-packages/yt_dlp/extractor/wakanim.py", line 50, in _real_extract
    group='url'))
  File "/usr/lib/python3/dist-packages/yt_dlp/extractor/common.py", line 1166, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
yt_dlp.utils.RegexNotFoundError: Unable to extract manifest url; please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/yt_dlp/extractor/common.py", line 595, in extract
    ie_result = self._real_extract(url)
  File "/usr/lib/python3/dist-packages/yt_dlp/extractor/wakanim.py", line 50, in _real_extract
    group='url'))
  File "/usr/lib/python3/dist-packages/yt_dlp/extractor/common.py", line 1166, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
yt_dlp.utils.RegexNotFoundError: Unable to extract manifest url; please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/yt_dlp/YoutubeDL.py", line 1311, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/yt_dlp/YoutubeDL.py", line 1336, in __extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python3/dist-packages/yt_dlp/extractor/common.py", line 621, in extract
    raise type(e)(e.msg, **kwargs)
yt_dlp.utils.RegexNotFoundError: [Wakanim] 27775: Unable to extract manifest url; please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.; please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
```

**Comment by pukkandan** [comment]
Incapsula is blocking the webpage download 😢

**Comment by Totorrr** [comment]
@pukkandan any cookie, referer or other parameter we could pass to Incapsula for it to allow the download you think?

**Comment by Yavos** [comment]
Not that it would matter anymore since they'll stop their service soon, but here's a solution to that Incapsula problem:

1. go to the website and start your developer tools in your favorite browser
2. open a page for an episode or movie
3. look in the network tab at the request of the actual website (first entry) and find request headers
4. there's a header called "`Cookie`" with pretty long content containing lots of `incap` and other stuff - copy that
5. add in command line to your request: `--add-headers "Cookie:<content>"` (in Firefox copying already included the `Cookie:` part)
6. also add `--cookies-from-browser `and `--user-agent` string from your browser, just in case (I only tested that way)
7. enjoy not downloading all the drm protected content (didn't have any luck so far - everything I tried has drm protection)
